src/index.html

```diff
@@ -11,6 +11,33 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="#"
+    class="big-form form"
+    data-qa="big">
```

Comment:
```suggestion
    <form
      action="#"
      class="big-form form"
      data-qa="big"
    >
```
fix attribute indents in all cases